files/conf/docker-daemon.json

```diff
@@ -1,5 +1,5 @@
 {
     "registry-mirrors": [
-        "https://quay.io"
+        "https://{{ .RegistryDomain }}"
```

Comment:
I have mixed feelings if we should configure that. What is the use case? If this is configured then it should be clear that this is the list of mirrors IMO. From variable description I'd try to set it to empty string to use dockerhub.

Reply:
Discussed on Slack https://gigantic.slack.com/archives/C010L893ZFW/p1592393492040000.

TL;TR: we need to make AWS China work.